tests/vhost_user_blk_test_server.c

```diff
@@ -1035,8 +1035,8 @@ static void monitor_serve_unix_socket(const char *path,
 
 static void interactive_sigint(int sig)
 {
-    const char *msg = "\nUse 'stop' command or Ctrl+D to exit\n";
-    write(2, msg, strlen(msg));
+    const char *const msg = "\nUse 'stop' command or Ctrl+D to exit\n";
+    (void)write(2, msg, strlen(msg));
```

Comment:
Last time I checked this didn't help against the `nodiscard` warning. Does it for you?

Reply:
it was tracing paper with https://github.com/yandex-cloud/yc-libvhost-server/blob/master/vdev.c#L1943
If it doesn’t make sense here, then I’ll remove it there too

Reply:
I guess it's fine to keep it as long as it helps get rid of a warning.